AAUnlimited/Functions/Shared/Triggers/Expressions.cpp

```diff
@@ -3019,6 +3113,19 @@ namespace Shared {
 					{ TYPE_STRING }, (TYPE_BOOL),
 					&Thread::PoseExists
 				},
+				{
+					44, EXPRCAT_EVENT,
+					TEXT("Npc Effective Response Success"), TEXT("EffectiveResponseSuccess"),
+					TEXT("If executed in a trigger with the After NPC Answers Event, this is the current answer that the NPC will act upon, modified by any previously executed Triggers."),
+					{}, (TYPE_BOOL),
+					&Thread::GetNpcResponseCurrentAnswerSuccess
+				},
+				{
+					45, EXPRCAT_CHARPROP,
+					TEXT("Preference"), TEXT("%p ::Preference( %p )"), TEXT(""),
```

Comment:
Specify that it's H preference. People might think characters have some inherent preferences, like "info fatsos"